pkg/gatewayserver/redis/registry_test.go

```diff
@@ -109,8 +109,30 @@ func TestRegistry(t *testing.T) {
 
 	t.Run("ClearManyTimes", func(t *testing.T) {
 		a, ctx := test.New(t)
-		a.So(registry.Set(ctx, ids, nil, nil), should.BeNil)
-		a.So(registry.Set(ctx, ids, nil, nil), should.BeNil)
+		a.So(registry.Set(ctx, ids, nil, nil, 0), should.BeNil)
+		a.So(registry.Set(ctx, ids, nil, nil, 0), should.BeNil)
+	})
+
+	t.Run("SetWithTTL", func(t *testing.T) {
+		a, ctx := test.New(t)
+		stats := &ttnpb.GatewayConnectionStats{
+			DisconnectedAt: ttnpb.ProtoTimePtr(time.Date(2021, 12, 02, 11, 24, 58, 0, time.UTC)),
+		}
+
+		err := registry.Set(ctx, ids, stats, []string{"disconnected_at"}, 2*time.Second)
```

Comment:
You can use smaller values, and checkout our usage of `test.Delay`. That one scales with the `TEST_SLOWDOWN` environment variable (we generally use 8).

Reply:
I didn't even notice it, it's a cool idea. Changed the test to use it.